features/guidance/turn-angles.feature

```diff
@@ -1189,7 +1189,7 @@ Feature: Simple Turns
 
         When I route I should get
             | waypoints | route             | turns                           |
-            | a,c       | rose,trift        | depart,arrive                   |
+            | a,c       | rose,trift,trift  | depart,turn slight left,arrive  |
```

Comment:
this changed due to traffic signals now being collapsed. Needed to fix #4332 test now